.changelog/337.internal.md

```diff
@@ -0,0 +1,8 @@
+Add Change Log and the Change Log fragments process for assembling it
```

Comment:
I would classify this under "Process Changes", i.e. renaming the snippet to `337.process.md`.